### PR TITLE
Follow-up edits to PR#3781

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/3781

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape PTAL. Thanks!

File: architecture/additional_concepts/contiv.adoc [hunk 1]
```diff
@@ -13,22 +13,43 @@ toc::[]
...
-- Configuring the network forwarding layer components for layer 2 or layer 3 forwarding
-- Configuring and enforcing a range of network policies that provide  
-- Providing management interfaces (including both CLI and GUI) to configure and manage contiv-specific features and configurations
-- Providing an infrastructure for role based controls that allow for multiple role based network operations workflows. 
+[[architecture-additional-concepts-contiv-architecture]]
+== Architecture
 
-*Contiv* uses the Container Network Interface (link:https://kubernetes.io/docs/admin/network-plugins/#cni[CNI]) to interface with Openshift/ Kubernetes. A key value store based on *etcd* is used to store contiv-specific state information. This is in addition to and separate from the instance of *etcd* used by other components in the system including Openshift itself. 
+Each node within the cluster runs a Contiv agent called *netplugin* while the
+master hosts run the Contiv controller (*netmaster*), along with supporting
```
Comment (adellape) [inline architecture/additional_concepts/contiv.adoc]: Maybe make the (netmaster) into a "called netmaser" style, to better match the "calling netplugin" before it.

**Review by adellape — APPROVED** [review]
LGTM

**Comment by ahardin-rh** [comment]
@rajatchopra PTAL. I added a NOTE indicating that configuration instructions are forthcoming. Is that accurate? Thanks!

**Comment by ahardin-rh** [comment]
Received confirmation and approval from @srampal 

Merging this now, then kicking off a build.

**Pull request merged**